internal/cli/atlas/deployments/setup.go

```diff
@@ -46,9 +46,12 @@ const (
 	clusterNamePattern = "^[a-zA-Z0-9][a-zA-Z0-9-]*$"
 )
 
+var ErrPodmanNotReady = errors.New("podman not found in your system, check requirements at http://docpage")
```

Comment:
will this be podman link itself of a DOP page if so maybe we can create the JIRA?

Reply:
I think this has to be a docsp page (ours)